mam4_refactor_scripts/test_gen_for_sub.sh

```diff
@@ -73,15 +87,39 @@ main() {
 #Generate a newline
 newline () { echo ''; }
 
+create_code_for_in_outs () {
+    newline
+    newline
+    echo 'Generating files with code to add to the include files ......'
+    newline
+    #call python script to generate code for the calls
+    MY_PATH="$(dirname -- "${BASH_SOURCE[0]}")" # relative path
+    MY_PATH="$(cd -- "$MY_PATH" && pwd)"        # absolute and normalized path
+    /bin/python3 $MY_PATH/gen_input_write_vars.py  --finp $3 --fout $4 -i "$1" -o "$2"
+
+    #check if the files exists that has intent-in, intent-inouts and intent-outs calls
+    if [ ! -f $3 ]; then
+        echo "File tmp_in_calls.txt doesn't exist, check gen_input_write_vars.py script, exiting"
+        exit 1
+    fi
+    if [ ! -f $4 ]; then
+        echo "File tmp_out_calls.txt doesn't exist, check gen_input_write_vars.py script, exiting"
+        exit 1
+    fi
+
+}
+
+
 create_file () {
 
     #create_file $module_name $sub_name $yaml_dir_path $relative_path "end"
```

Comment:
In above comment could change ' "end" ' to ' "beginning"/"end" '.

Reply:
Actually, I guess ' $ftmp_ins/$ftmp_outs ' also would need to be inserted after ' $relative_path '.